pages/glossary/visible-inner-text.md

```diff
@@ -0,0 +1,38 @@
+# Visible inner text
```

Comment:
This definition needs a frontmatter.

Reply:
Thanks again, I just pushed a commit for this.